### PR TITLE
fix: drop music misc feedback

### PR DESCRIPTION
**Pull request by alantoa**
# Why

just quickly fix and update much drop-music feedback on the slack channel.

### CODE REVIEW

File: packages/app/components/drop/drop-event.tsx [hunk 1]
```diff
@@ -554,21 +554,17 @@ export const DropEvent = () => {
                     </View>
                     <ScrollView tw="flex-row" horizontal={true}>
                       <DataPill
-                        label={`Royalties ${watch("royalty")}%`}
+                        label={`${watch("editionSize")} Editions`}
```
Comment (hirbod) [inline packages/app/components/drop/drop-event.tsx]: This does not take into account when there is 1 Edition, 2 Edition**s**
Singular/Plural

File: packages/app/components/drop/drop-free.tsx [hunk 2]
```diff
@@ -502,17 +502,17 @@ export const DropFree = () => {
                     </View>
                     <ScrollView tw="flex-row" horizontal={true}>
                       <DataPill
-                        label={`Royalties ${watch("royalty")}%`}
+                        label={`${watch("editionSize")} Editions`}
```
Comment (hirbod) [inline packages/app/components/drop/drop-free.tsx]: Singular/Plural (1 Edition vs 10 Editions)

File: packages/app/components/drop/drop-private.tsx [hunk 3]
```diff
@@ -527,17 +527,18 @@ export const DropPrivate = () => {
                     </View>
                     <ScrollView tw="flex-row" horizontal={true}>
                       <DataPill
-                        label={`Royalties ${watch("royalty")}%`}
+                        label={`${watch("editionSize")} Editions`}
```
Comment (hirbod) [inline packages/app/components/drop/drop-private.tsx]: singular/plural

**Review by hirbod — COMMENTED** [review]
LGTM, just had one comment about singular and plural for editions.

File: packages/app/components/drop/drop-event.tsx (hunk 1 above)
Comment (alantoa) [reply packages/app/components/drop/drop-event.tsx]: 👍 yeah, I was though it too.

Comment (alantoa) [reply packages/app/components/drop/drop-event.tsx]: just fixed and update it, thanks remind me.

**Pull request merged**